main.py

```diff
@@ -0,0 +1,61 @@
+"""Worker to extract Indonesia Trending YouTube Video Statistics."""
+import os
+import logging
+from datetime import datetime
+from time import time
+
+import pandas as pd
+
+from ytid import config, YouTube
+from ytid.logger import setup_logging
+
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def save_to_csv(data, filename="trending.csv"):
+    """Save data as csv in the given datadir."""
+    if not os.path.isdir(filename.split("/")[-2]):
+        os.makedirs(filename.split("/")[-2])
+
+    if not os.path.exists(filename):
+        data.to_csv(filename, index=False)
+        return
+
+    data.to_csv(filename, index=False, header=False, mode="a")
+    return
```

Comment:
It's better to put this utility function inside a utility module, e.g. `common/utils.py`.